src/apps/store/controllers/book.controller.ts

```diff
@@ -13,15 +13,17 @@ import { BookCreateDto } from '@apps/store/dtos/book/create.dto';
 import { BookUpdateDto } from '@apps/store/dtos/book/update.dto';
 import { BookService } from '@apps/store/services/book.service';
 import { PaginateType, UserRole } from '@shared/type';
+import { BookSearchDto } from '@apps/store/dtos/book/search.dto';
 
 @Service()
 @JsonController('/book')
 export class BookController {
   constructor(private readonly bookService: BookService) {}
 
   @Get('/all')
-  async getAll(@QueryParams() params: PaginateType) {
-    return this.bookService.getAll(params);
+  async getAll(@QueryParams() params: BookSearchDto & PaginateType) {
```

Comment:
Let's extends `PaginateDto` from `BookSearchDto` instead of declaring like this. Because this way didn't include the validation stuff.